cookbook/ui_customization/add_a_tab_to_product_edit_form.rst

```diff
@@ -0,0 +1,212 @@
+How to add a tab or a panel to the product edit form
+====================================================
+
+.. note::
+    #Add a note to the "How to work with the front in the PIM"
+
+The most common UI customization on the Akeneo PIM is to add a tab to the product edit form. With the new product edit form introduced in 1.4 (more details here#TODO add link) we splitted tabs in two concepts:
+
+ - Tabs hold all edit features on the entity (attributes edit, classification, associations)
+ - Panels hold all meta information about the entity (history, completeness, comments)
```

Comment:
Don't you have a picture to summarize this visually? (coming from you talk about PEF)